pkgs/development/python-modules/async-tkinker-loop/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, pkgs
+, python3
+, fetchPypi
+}:
+python3.pkgs.buildPythonPackage rec {
+  pname = "async-tkinker-loop";
+  version = "0.8.1";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "async_tkinter_loop";
+    hash = "sha256-+9AvnYIZMWCbpCEKdbIadyU8zgyUlW/fRYYyDOxAzeg=";
+  };
+
+  buildInputs = with python3.pkgs; [
+    poetry-core
+  ];
+
+  patches = [
+    ./asyncio-built-in.patch
+  ];
```

Comment:
Would you please use `pythonRemoveDeps`?

Reply:
I took a stab at this, but it does not seem to have an effect on the poetry-defined dependencies? Perhaps I used it wrong - how are you suggesting I apply it?

Reply:
First, add `pythonRelaxDepsHook` to `nativeBuildInputs`.
```nix
 nativeBuildInputs = [
    pythonRelaxDepsHook
  ];
```

Next, we can use `pythonRemoveDeps`.
```nix
  pythonRemoveDeps = [
    "asyncio"
  ];
```
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonrelaxdepshook-using-pythonrelaxdepshook

Reply:
Thanks :) This unfortunately does not have an effect. The approach you describe here is what I also tried initially and following the given link I also tried setting `pythonRemoveDeps` with equally little effect.

Reply:
Which `pname` are you referring to? The `pname` of buildPythonPackage already has that value. The `pname` of `fetchPypi` is set to "async_tkinter_loop" because that is the name of the package on pypi - if we change that then the fetch will fail. Is there a third `pname` you're referring to should be explicitly set somewhere which is currently being inherited?

Reply:
Ah! Just now I realized you were referring to a typo in there rather than the difference in `-` vs `_` :D

Reply:
I should have pointed out that it was a typo.
Sorry for the confusion.

Reply:
Excellent! Everything now works on my end without the patch. Thank you so much for your patience :)